### DISCUSSION TITLE
sharing: [twitter, linkedin] not working as expected

### DISCUSSION THREAD

**Discussion by ICTINUS-Engenharia** [quarto-dev/quarto-cli#8951] [Q&A]
### Bug description

I've set up sharing: [twitter, linkedin] in _quarto.yml but whenever I click on share button on top right corner I got a error message. I figure out that url is  not passed to sharing command as you can see i get the link below when I hover my mouse over share button.

Example: twitter

https://twitter.com/intent/tweet?url=|url|

You can check it out , see site https://ictinus.com.br/

What am I doing wronge ?

I'm using Quarto version **1.3.450**

<img width="960" alt="quarto error" src="https://github.com/quarto-dev/quarto-cli/assets/33582443/e66eaaad-15c3-463e-bb97-b3432b4154fd">
<img width="956" alt="Quarto share error" src="https://github.com/quarto-dev/quarto-cli/assets/33582443/673c4f72-301e-41a6-8451-6c68d8e2091e">

### Your environment

VS-Studio: 1.87.0

**Comment by mcanouil** [comment]
> What am I doing wronge ?

No code, no help unfortunately as we cannot guess your setup from a screenshot.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by ICTINUS-Engenharia** [reply]
see below my _quarto.yml. Why it does not grab my page link to share in social media ? I mean, sharing: [twitter, linkeding] not working.

project:
  type: book
  output-dir: _book
  preview:
    browser: true
    navigate: true
  render:
    - "*.qmd"

title-prefix: "noprefix"

website:
  title: ICTINUS
  sidebar:
    - title: "Home"
      style: "docked"
      background: light
      contents:
        - index.qmd

  comments:
    hypothesis:
      theme: clean
      openSidebar: true

  cookie-consent:
    type: express
    style: headline
    palette: dark

  # google analytics
  #google-analytics:
  #  tracking-id: "G-M21L0CBCVN"
  #  anonymize-ip: true
  #  version: 4

book:
  navbar:
    search: true
    pinned: false
    collapse: true
    back-to-top-navigation: true
    left:
      - text: Home
        href: index.qmd
      - text: More
        menu:
          - text: "Livro Grafcet"
            href: "https://ictinus.com.br/grafcet/index.html"
    right:
      - icon: instagram
        href: https://www.instagram.com/ictinusengenharia/
      - icon: facebook
        href: https://www.facebook.com/ictinusengenharia

  sidebar:
    search: true
    collapse-level: 2
    border: false
    pinned: false
    tools:
      - icon: instagram
        href: https://www.instagram.com/ictinusengenharia/
      - icon: facebook
        href: https://www.facebook.com/ictinusengenharia

  page-navigation: true
  title: "ICTINUS"
  subtitle: "Solução em Automação Industrial e Engenharia"
  abstract: "Somos a **{{< var title.long >}}**, uma empresa voltada para soluções em Engenharia e Automação Industrial, aplicando boas práticas em Automação para encontrar a melhor solução para sua necessidade. Temos parcerias com os mais conhecidos fabricantes."
  author: "Eng. Luciano Cardoso Vasconcelos"
  search: true
  site-url: https://ictinus.com.br/index.html
  repo-url: https://github.com/ICTINUS-Engenharia/ictinus
  # repo-actions: [edit, issue, source]
  repo-actions: [issue]
  sharing: [twitter, linkedin]
  reader-mode: true
  cover-image: coverGrafcet.png
  favicon: favicon.png
  page-footer:
    left: "por Eng. Luciano Vasconcelos (CEO Ictinus Engenharia)"
    center:
      - text: "Quarto"
        url: "https://quarto.org/"
    right:
      - icon: instagram
        url: https://www.instagram.com/ictinusengenharia/
      - icon: facebook
        url: https://www.facebook.com/ictinusengenharia

  chapters:
    - text: "---"
    - part: ACONDICIONAMENTO
      chapters:
        - index.qmd
        - contents/dedication.qmd
    - text: "---"
    - part: FABRICAÇÃO
      chapters:
        - contents/introduction.qmd
    - text: "---"
    #  - part: REFERENCES
    #    chapters:
    #      - references.qmd
    #  - text: "---"
    #- part: EXERCÍCIOS
    #  chapters:
    #    - contents/niclav_sys/niclav_sys.qmd
    ##  - contents/image_classification/image_classification.qmd
    ##  - contents/object_detection_fomo/object_detection_fomo.qmd
    ##  - contents/kws_feature_eng/kws_feature_eng.qmd
    ##  - contents/kws_nicla/kws_nicla.qmd
    ##  - contents/dsp_spectral_features_block/dsp_spectral_features_block.qmd
    ##  - contents/motion_classify_ad/motion_classify_ad.qmd
    #- text: "---"
    - part: ANEXOS
      chapters:
        - contents/anxFontes.qmd

  appendices:
    - contents/tools.qmd
    - contents/bConfiguração/bConfiguração.qmd
  #  - contents/zoo_datasets.qmd
  #  - contents/zoo_models.qmd
  #  - contents/learning_resources.qmd
  #  - contents/community.qmd
  #  - contents/case_studies.qmd

citation: true

reference-location: margin
citation-location: margin

keywords:
  - grafcet
  - PLC
  - STL
  - ICTINUS
  - ladder
  - Simatic
  - TIA Portal

#license: CC-BY-NC-SA

crossref:
  appendix-title: "Apêndice"
  appendix-delim: ":"

format:
  html:
    theme:
      light:
        - cosmo
        - style.scss
        - style-light.scss
      dark:
        - darkly
        - style.scss
        - style-dark.scss
    mainfont: Nunito
    highlight-style: github
    code-link: true
    link-external-icon: false
    link-external-newwindow: true
    callout-appearance: simple
    anchor-sections: true
    smooth-scroll: false
    citations-hover: false
    footnotes-hover: false
    fig-width: 8
    fig-height: 6
    number-depth: 3
    toc: true
    toc-depth: 4
  #  include-in-header:
  #text: <script async src="https://www.googletagmanager.com/gtag/js?id=G-M21L0CBCVN"></script>

editor:
  render-on-save: true

**Reply by mcanouil** [reply]
Please take two seconds to look at what you posted and look at the example I shared.
We cannot use that.

Also, to share multiple documents/files, it is best to do so via a Git repository.

Additionally and that is important, please make it small and reproducible, we unfortunately don't have the time to remove each option one at a time to figure out the issue.


---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
